test/parallel/test-dgram-exclusive-implicit-bind.js

```diff
@@ -96,4 +99,12 @@ if (process.env.BOUND === 'y') {
   source.unref();
 }
 
-source.send(Buffer.from('abc'), 0, 3, common.PORT, '127.0.0.1');
+function send() {
```

Comment:
I was concerned that competing 1ms interval timers on some operating systems could result in increased flakiness, but I didn't actually test it, so yeah, let's try that. :-D